interactive/emails.py

```diff
@@ -0,0 +1,21 @@
+from django.conf import settings
+from furl import furl
+from incuna_mail import send
+
+
+def send_report_uploaded_notification(analysis_request):
+    analysis_request_url = furl(settings.BASE_URL) / analysis_request.get_absolute_url()
+
+    context = {
+        "name": analysis_request.created_by.name,
+        "title": analysis_request.title,
+        "url": analysis_request_url,
+    }
+
+    send(
+        to=analysis_request.created_by.email,
+        sender="notifications@jobs.opensafely.org",
+        subject=f"A report has been released for your interactive study: {analysis_request.title}",
```

Comment:
The word "released" is quite Job Server specific, I'm not sure Interactive users will know what that means. Also, being extremely picky, I'm not sure Interactive users will be doing "studies" either, although that's a conversation for another time.

How about this for the subject: "Your OpenSAFELY Interactive report is ready to view"